web/src/topic_popover.js

```diff
@@ -63,6 +63,10 @@ export function initialize() {
                     return;
                 }
 
+                if (instance.context.is_topic_empty) {
+                    return;
+                }
+
```

Comment:
Was the need for this guard statement due to the ClipboardJS initialization below?